File: data/andrewshitov/Dart/Challenge89Part1.dart
void main(List args) {
    var n = 3;
    if (args.length == 1)
        n = int.parse(args[0]);

    var s = 0;
    for (var x = 1; x <= n; x++) {
        for (var y = x + 1; y <= n; y++) {
            s += x.gcd(y);
        }
    }

    print(s);
}

File: data/andrewshitov/Perl/Challenge89Part1.pl
use v5.20;

use Math::Utils qw(gcd);

my $n = $ARGV[0] // 3;

my $s = 0;
for my $x (1 .. $n) {
    for my $y ($x + 1 .. $n) {
        $s += gcd($x, $y);
    }
}

say $s;